test/java/org/opendatakit/briefcase/util/BriefcaseVersionManagerTest.java

```diff
@@ -57,4 +57,18 @@ public void knows_if_we_are_not_up_to_date() {
 
     assertThat(versionManager.isUpToDate(), is(false));
   }
+
+  @Test
+  public void version_contains_hyphen(){
```

Comment:
This test is useful. However, I think the test method requires a bit of work. Maybe something along the lines "ignores beta version information"